src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_api.py

```diff
@@ -969,19 +1066,81 @@ class PROCESSENTRY32(ctypes.Structure):
                     ("szExeFile", ctypes.c_char * 260)]
```

Comment:
I think the problem was here, actually. Here's the corresponding C struct:
```c
typedef struct tagPROCESSENTRY32 {
  DWORD     dwSize;
  DWORD     cntUsage;
  DWORD     th32ProcessID;
  ULONG_PTR th32DefaultHeapID;
  DWORD     th32ModuleID;
  DWORD     cntThreads;
  DWORD     th32ParentProcessID;
  LONG      pcPriClassBase;
  DWORD     dwFlags;
  CHAR      szExeFile[MAX_PATH];
} PROCESSENTRY32;
```
Note that `th32DefaultHeapID` is ULONG_PTR, which means that it's 64 bits in a 64-bit process.  But `ctypes.c_ulong` is always going to be 32-bit on Windows. It should be `c_size_t`.

I would also change all DWORD members to `c_uint32`, since that's what the typedef is meant to represent.

Reply:
Let me try this change locally and see if that fixes it...

Reply:
Yep, now it works! Let me make for that.